recipe/meta.yaml

```diff
@@ -37,11 +37,11 @@ requirements:
     - dvc-http >=2.29.0
     - dvc-objects
     - dvc-render >=1.0.1,<2
-    - dvc-studio-client >=0.20,<1
+    - dvc-studio-client >=0.21,<1
     - dvc-task >=0.3.0,<1
     - flatten-dict <1,>=0.4.1
-    - flufl.lock >=5,<8
-    - fsspec
+    - flufl.lock >=8.1.0,<9
```

Comment:
Requires an update in https://github.com/conda-forge/flufl.lock-feedstock to get `flufl.lock=8.1.0`.

Reply:
Pushing for an update at https://github.com/conda-forge/flufl.lock-feedstock/pull/14. @efiop, could you review that PR since you're a maintainer for that recipe feedstock?

Reply:
Hey @weiji14 , the tests are failing in that PR though. I'm happy to add you in there as well as a maintainer, just let me know.

Reply:
The fix on that PR should be staightforward, I've mentioned at https://github.com/conda-forge/flufl.lock-feedstock/pull/14/files#r1694383020 that it involves changing `pdm-backend` -> `hatchling`. I'd prefer not to be a recipe maintainer there as there's too many (20+) packages on my plate already :slightly_smiling_face:

Reply:
@weiji14 No worries, I totally understand that 🙂 Adjusted the PR and merged it.

Reply:
thanks @efiop @weiji14 !